src/software/ai/hl/stp/play/ball_placement/ball_placement_play_test.py

```diff
@@ -14,9 +14,9 @@
 
 
 def ball_placement_play_setup(
-    ball_start_point, ball_placement_point, simulated_test_runner
+    ball_start_point, ball_placement_point, simulated_test_runner, blue_only
 ):
```

Comment:
nit: would be nice if we added a docstring for this function just so that we're consistent